.NET/Microsoft.Recognizers.Text.DateTime/Extractors/IDateTimeExtractor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Enable a reference time pass to the extractor
```

Comment:
This should be deleted since we do the IDatetimeExtractor in another PR.

Reply:
I didn't see it in my project, maybe it still exist in folder, do I need to delete it in folder also?